lib/puppet-strings/yard.rb

```diff
@@ -114,18 +114,18 @@ def output(name, data, undoc = nil)
     @undocumented += undoc if undoc.is_a?(Integer)
     data =
       if undoc
-        ('%5s (% 5d undocumented)' % [data, undoc])
+        "#{data} (#{undoc} undocumented)"
```

Comment:
This is what the new output will look like if there is more than 9 of some type (assuming I didn’t read this wrong):

```
Files:                    1
Modules:                  0 (0 undocumented)
Classes:                  0 (0 undocumented)
Constants:                0 (0 undocumented)
Attributes:               0 (0 undocumented)
Methods:                  0 (0 undocumented)
Puppet Classes:           10 (0 undocumented)
Puppet Data Types:        0 (0 undocumented)
Puppet Data Type Aliases: 10 (10 undocumented)
Puppet Defined Types:     1 (0 undocumented)
Puppet Types:             0 (0 undocumented)
Puppet Functions:         0 (0 undocumented)
Puppet Providers:         0 (0 undocumented)
Puppet Plans:             0 (0 undocumented)
Puppet Tasks:             0 (0 undocumented)
```

I think that’s fine, especially since I imagine the vast majority of modules have fewer than <10 of anything. Just wanted to make it explicit.